src/views/Trace/Actions/ToolActions/index.js

```diff
@@ -11,16 +11,19 @@ import applicationActions from 'ducks/application/actions';
 const TracerToolActions = () => {
   const dispatch = useDispatch();
   const points = useSelector(state => pointsSelectors.getPoints(state));
+  const filteredPoints = useSelector(state => pointsSelectors.getFilteredPoints(state));
+
+  const allowStaging = points.length === filteredPoints.length;
```

Comment:
Currently the logic for displaying the `Clear all filters` button in the SelectedData component also checks to see if any points are hidden, but from my understanding that addition is redundant since filteredPoints already filters out any hidden points.

Let me know if I'm missing something